### DISCUSSION TITLE
有没有交流群

### DISCUSSION THREAD

**Discussion by ridgeback1982** [Tencent/libpag#2542] [Q&A] [answered]
微信群或者qq群，可以讨论些开发问题的。

**Comment by ridgeback1982** [comment]
有没有人想把libPAG改成视频编辑软件的引擎的，我感觉可行，希望有人一起讨论下 :-)。

**Reply by kevingpqi123** [reply, marked as answer]
建议系统看下官网的文档，pag 诞生之初就是为了解决视频编辑场景的复杂特效渲染问题，重点看下应用场景
https://pag.io/docs/faq.html#%E9%97%AEpag-%E4%B8%8E%E8%A7%86%E9%A2%91%E7%BC%96%E8%BE%91%E7%9A%84%E5%85%B3%E7%B3%BB

视频后编辑的视频模版、直播场景的特效都可以使用 pag 来实现，大家在微信中看到的游戏战报使用的都是 pag 的渲染方案

**Reply by ridgeback1982** [reply]
因为libPAG解决了渲染的空间层级和时间线的问题，所以我理解再加上一个音频混合和媒体编解码，基本就是一个视频剪辑引擎了。TAVMedia就是这么一个引擎，但是不开源。我不清楚TAVMedia的实现框架是什么。我想问，TAVMedia有没有部分参考libPAG（比如说使用libPAG的渲染树），还是说TAVMedia是完全另外实现的（支持pag文件作为一个素材源）？
